private_gpt/ui/ui.py

```diff
@@ -154,6 +157,32 @@ def build_history() -> list[ChatMessage]:
                     for index, source in enumerate(sources, start=1)
                 )
 
+    # On initialization and on mode change, this function set the system prompt
+    # to the default prompt based on the mode (and user settings).
+    @staticmethod
+    def _get_default_system_prompt(mode: str) -> str:
+        p = ""
+        match mode:
+            case "Query Docs":
+                p = "You can only answer questions about the provided context. If you know the answer " \
```

Comment:
Thanks... The challenge is you may want 2 separate settings fields (1 for default_query_system_prompt, and 1 for default_chat_system_prompt), since the default prompts for those vary. 

I can go ahead and make those 2 default prompts explicitly defined in the settings, but the query prompt has no other default defined (unlike chat prompt, which can use `llama_index.DEFAULT_SYSTEM_PROMPT`).

Do you recommend we provide the default query prompt in the settings.yaml, and have it as a required field, or should we hard code a default value in the code, and allow users to override via settings?

Reply:
I've pushed the solution above for testing (2 different settings fields) for evaluation. I'm open to another approach if there's a more efficient or clear solution. Thanks!